src/database/migrations/20200205141154-create-accomodation.js

```diff
@@ -45,18 +44,16 @@ export default {
     status: {
       type: Sequelize.STRING,
       allowNull: false,
-      defaultValue: 'Pending',
+      defaultValue: 'Pending'
     },
     createdAt: {
       allowNull: false,
-      type: Sequelize.DATE,
-      defaultValue: Sequelize.fn('NOW')
+      type: Sequelize.DATE
     },
     updatedAt: {
       allowNull: false,
-      type: Sequelize.DATE,
-      defaultValue: Sequelize.fn('NOW')
```

Comment:
Why did you delete the default value constraint?

Reply:
I have fixed issue.